client/src/components/graph/graph.tsx

```diff
@@ -98,7 +100,15 @@ type GraphState = {
   };
   modelTF: ReadonlyMat3;
   modelInvTF: ReadonlyMat3;
+  testImageSrc: string | null;
```

Comment:
maybe worth adding a comment to explain the context on why we need this prop for future reference? Something along the line that "this is used for Chromatic visual test` or something better!